MantidQt/CustomInterfaces/test/ReflSettingsTabPresenterTest.h

```diff
@@ -217,6 +217,7 @@ class ReflSettingsTabPresenterTest : public CxxTest::TestSuite {
         .WillOnce(Return("INTER"));
     EXPECT_CALL(mockView, setInstDefaults(defaults)).Times(1);
     presenter.notify(IReflSettingsTabPresenter::InstDefaultsFlag);
+    TS_ASSERT(Mock::VerifyAndClearExpectations(&mockView));
```

Comment:
Thanks for adding this. I think the test `testExperimentDefaults` is also missing this line.